### PR TITLE
Run prettier over codebase

### PR DESCRIPTION
**Pull request by munroTom**

### CODE REVIEW

File: scripts/syncLegacyDB.js [hunk 1]
```diff
@@ -47,36 +47,36 @@ db.settings(settings);
...
           const location = new admin.firestore.GeoPoint(latitude, longitude);
 
           photo.location = location;
-          searchPhase = 'url'
+          searchPhase = "url";
         }
         break;
 
-      case 'url':
-        if (line.match("popup\.setContent")) {
```
Comment (sebastianovide) [inline scripts/syncLegacyDB.js]: @munroTom  Are you sure it won't break the Regex ?

File: scripts/syncLegacyDB.js [hunk 2]
```diff
@@ -47,36 +47,36 @@ db.settings(settings);
...
-          photo.url = line.match("(https?://.*?)\'>")[1];
```
Comment (sebastianovide) [inline scripts/syncLegacyDB.js]: need Regex test

File: scripts/syncLegacyDB.js [hunk 3]
```diff
@@ -86,69 +86,84 @@ async function getPhotosFromMap() {
           if (dateTime) {
             const date = dateTime[1];
             const time = Number(dateTime[2]);
-            photo.updated = new Date((new Date(date)).getTime() + time)
+            photo.updated = new Date(new Date(date).getTime() + time);
           }
 
           // google
           if (!photo.updated) {
-            const time = line.match("_(\\d{10,})\.jpg");
```
Comment (sebastianovide) [inline scripts/syncLegacyDB.js]: "." means any character while "\." is would match a a dot.

File: scripts/syncLegacyDB.js (hunk 1 above)
Comment (munroTom) [reply scripts/syncLegacyDB.js]: @sebastianovide I could leave it out of this file for now as it seems to be a script that's run separately?
 I had a click through the app after I did it and then did some more work over the weekend and everything seemed to work as expected. I just found it easier cause I like using prettier and it means that I can keep track of the changes made a bit easier in the diffs if prettier is run over everything in one go

**Comment by sebastianovide** [comment]
@munroTom conflicts

**Comment by munroTom** [comment]
@sebastianovide after the next lot of work has gone in I'd like to run prettier over the codebase again. I can leave it out of the scripts file where you found potential issues from untested regex in the `syncLegacyDB` script but I'm finding that I'm spending more time trying to keep the diffs of commits small because I'm touching files I haven't before, sometimes forgetting to switch prettier off and then having to go back and try to fix the diff size because I've change an import path (or similar small changes)
If there were an enforced code style throughout the app then I would happily follow it but I can't find consistent evidence of one, I've attached a few screenshots below of code snippets to help make that point

Semi colon and no semi colon
Screen Shot 2019-10-14 at 18 31 29
Single and double quotes
Screen Shot 2019-10-14 at 18 31 39
Trailing comma and no trailing comma
Screen Shot 2019-10-14 at 18 33 17

Especially with this being a project where new developers may be coming and going more frequently a formatting tool that helps to ensure uniformity between all of them would speed up developing time and improve the experience for those unfamiliar with the codebase

**Pull request merged**